pkg/operator/minio/controller.go

```diff
@@ -43,6 +43,7 @@ const (
 	customResourceNamePlural    = "objectstores"
 	minioCtrName                = "minio"
 	minioLabel                  = "minio"
+	minioObjectStoreLabel       = "minio_objectstore"
```

Comment:
What if this label were just `objectstore`? If they wanted to guarantee minio, then their query would be `app=minio,objectstore=my-store`. Otherwise, if they know they only have minio, they could simplify with `objectstore=my-store`. Either way is fine though.